README.md

```diff
@@ -82,6 +82,16 @@ Also, additional utils from `sources/utils/` are used in tests and deployment sc
 
 `tact.config.json` contains the configuration for the Tact compiler. In most cases, you don't need to modify this file.
 
+## Smart-Contracts Structure
+
+Main smart contract is `jetton_minter.tact`. It imports `messages.tact` and `jetton_wallet.tact`, so they will be compiled automatically, when setting `jetton_minter.tact` as target in `tact.config.json`.
+### Traits
+Jetton minter is using *OwnableTransferable* and *Deployable* trait. Actually, you can remove *Deployable* trait. It is used only for more convenient deployment in tests. 
```

Comment:
> Actually, you can remove *Deployable* trait.

Looks like this only contributes to the code size growth, so let's remove it from the code and README

Reply:
Well, that is for tests to work. But okay, I will deploy the contract without deployable trait (But this may look like a magic to newcomers)

Reply:
> But this may look like a magic to newcomers

But then we should explain it!